fetch.bs

```diff
@@ -307,9 +307,30 @@ and optionally an <var>extract-value flag</var>, run these steps:
  inclusive, within <var>input</var>.
 </ol>
 
-<p class="note no-backref">The <var>extract-value flag</var> argument makes this algorithm suitable
-for <a for="header list">getting, decoding, and splitting</a> and <a>parse a MIME type</a>, as well
-as other header value parsers that might need this.
+<div class=example id=example-http-quoted-string>
+ <table>
+  <tr>
+   <th>Input
+   <th>Final position (always starts at 0)
```

Comment:
```suggestion
   <th>Final position (start position is always 0)
```

Reply:
Although it might be better to just delete this column... it's not an output of the algorithm, and it's not clear to me what it's helping to illustrate.

Reply:
It's an "outparam", no?

Reply:
Oh, right, that is important... hmm. Then I'd phrase it as something like "New position variable value", maybe?

Although I'm not sure readers are really going to manually count. It might be good to add a parenthetical value, e.g. 2 (end of string), 7 ( )? Or maybe a `<mark>` around the pointed-to-character? Meh, nothing works great there...

Reply:
https://whatpr.org/fetch/1069.html#example-http-quoted-string looks reasonable to me. What do you think?

Reply:
Yeah, that's pretty solid; ship it!